### DISCUSSION TITLE
Organization That You are The Owner: Will The Stars Still Count?

### DISCUSSION THREAD

**Discussion by jfullstackdev** [anuraghazra/github-readme-stats#1006] [Q&A] [answered]
I just want to ask because I am not so sure with the answer before I transfer my repos to organizations that I will create, so that it will be organized by categories (even though I'm working alone, I find it hard to get the overview of everything).

So my question is, If I transferred my repos to my own newly created organizations, will I lose the count of stars? And, when news stars are gained, will these stars be counted as part of my GitHub Stats? 

Thanks.

**Comment by jfullstackdev** [comment]
I just confirmed it because I simulated it, it will not be counted as part of your GitHub Stars because it belongs to an organization that is shared by users, even though in reality, for example, you created and own the repo and has no other members. 

That's the problem I will be losing the stars count not the actual stars if I will be organizing my projects into several GitHub Organization.

I also tried to make a card for an organization, it is not allowed.

**Comment by developStorm** [comment, marked as answer]
There is a long discussion about your issue in #1

**Comment by Hazrat-Ali9** [comment]
The stars may stop counting, but dreams never will.
